data/mm/world/overworld.yml

```diff
@@ -340,6 +356,8 @@
     "Swamp Front": "true"
   events:
     KOUME: "has_red_or_blue_potion"
+    STICKS: "true"
+    "NUTS": "true"
```

Comment:
No need to quote events

Reply:
Yeah this is fixed in the coming commit. i noticed it.